hcp-consul-hcp-vault-ca/provider.tf

```diff
@@ -0,0 +1,26 @@
+terraform {
+  required_version = ">= 1.1.5"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "3.74.0"
+    }
+    hcp = {
+      source  = "hashicorp/hcp"
+      version = "0.22.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = var.region
+  default_tags {
+    tags = var.default_tags
+  }
+}
+
+provider "hcp" {
```

Comment:
Rather than setting the credentials in a file (that could accidentally be committed), we recommend setting environment variables instead (similar to how you're doing it for the AWS provider). Check out this tutorial for some language and format you can use!
https://learn.hashicorp.com/tutorials/cloud/consul-ecs-hcp?in=consul/cloud-production#configure-aws-and-hcp-environment-variables

Reply:
Thanks @eddie-rowe ! I'm in the process of removing the file-based instructions from the tutorial and code 👍